ucsb-cs48-s20/project-s1-t4-missile
# Username entry

In this PR:
* made a username selection page that displays before the user is allowed into the lobby
* updated styles for username selection page and lobby
* added scrollable display for everyone in the lobby based on their username
* refactored index.js to gamepage.js, used React function component to keep code consistent
* removed parentGameWindow.js

To Test:
* enter a username in the title page, should bring you to the page with the game and chat component, which behaves in the same way as previously on master
* entering lobby adds your name to the list of active users
* exiting the lobby by closing the tab should remove your name from the list of active users
